integration_tests/pages/bookedtoday/arrivals/confirmArrival/confirmAddedToRoll.ts

```diff
@@ -9,7 +13,22 @@ export default class ConfirmAddedToRollPage extends Page {
 
   confirmationParagraph = (): PageElement => cy.get(`[data-qa=confirmation-paragraph]`)
 
-  locationParagraph = (): PageElement => cy.get(`[data-qa=location-paragraph]`)
+  details = ({
+    prison,
+    locationName,
+    prisonNumber,
+    name,
+  }: {
+    prison: string
+    locationName: string
+    prisonNumber: string
+    name: string
+  }) => {
+    cy.get(paragraph).contains(prison)
+    cy.get(location).contains(locationName)
+    cy.get(banner).contains(prisonNumber)
+    cy.get(banner).contains(name)
+  }
```

Comment:
must be a nicer way to do this rather than getting the elements multiple times - cypress lint doesn't like assigning to variables so think need to use promise callback but then it gives you a jquery element rather than cypress element  - nevertheless this works and is equivalent to what was being called before